methods/main_methods.py

```diff
@@ -100,26 +99,71 @@ def show_outliers(data, outliers, height_index=HEIGHT_INDEX):
     pyplot.show()
 
 ## Frequency removal
-def apply_frequency_removal(data):
+def apply_frequency_removal(data, compression_percent=0.25):
```

Comment:
Applies low frequency removal from data. Compression_percent (between 0 and 1) dictates how much compression needs to be done. @wesleyktatum